mp/src/materialsystem/stdshaders/pbr_dx9.cpp

```diff
@@ -68,6 +79,10 @@ BEGIN_VS_SHADER(PBR, "PBR shader")
 		SHADER_PARAM(BUMPMAP, SHADER_PARAM_TYPE_TEXTURE, "", "Normal texture");
         SHADER_PARAM(USEENVAMBIENT, SHADER_PARAM_TYPE_BOOL, "0", "Use the cubemaps to compute ambient light.");
 		SHADER_PARAM(SPECULARTEXTURE, SHADER_PARAM_TYPE_TEXTURE, "", "Specular F0 RGB map");
+#ifdef PBRPARALLAX
+		SHADER_PARAM(PARALLAX, SHADER_PARAM_TYPE_BOOL, "0", "Use Parallax Occlusion Mapping.");
+		SHADER_PARAM(PARALLAXHEIGHT, SHADER_PARAM_TYPE_FLOAT, "0.5", "Maximum Height of the Parallax Map");
```

Comment:
Is 0.5 a good default?

Reply:
maybe too high, depends on the heightmap. i want 0.5 since when you toggle $parallax on, you can see results immediately even without modifying the $parallaxheight ...